tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -23,35 +23,52 @@
 {% endblock %}
 
 {% block content %}
-    {% include "patterns/molecules/title-block/title-block.html" with item=page tags=False %}
-    {% pageurl page.blog_index as blog_index_url %}
+    <div class="grid grid--spacer-large streamfield">
+        <div class="authors">
+            {% for page_author in page.authors.all %}
+                {% include "patterns/molecules/author/author.html" with author=page_author.author item=page %}
+            {% endfor %}
+        </div>
 
-    {% for page_author in page.authors.all %}
-        {% include "patterns/molecules/author/author.html" with author=page_author.author %}
-    {% endfor %}
+        <div class="streamfield__title">
+            <h1 class="heading heading--one">{{ page.title }}</h1>
 
-    <div>
-        {% for tag in page.services %}
-            {% include "patterns/atoms/tag/tag.html" with item=tag small=True tag_link_base=blog_index_url %}
-        {% endfor %}
-    </div>
+            {% pageurl page.blog_index as blog_index_url %}
 
-    <div>
-        <time datetime="{{ item.date|date:'Y-m-d' }}">{{ page.date }}</time>
+            <div class="tags tags--with-title">
+                <span class="sr-only">Filter blog posts by category</span>
+                {% for tag in page.services %}
+                    {% include "patterns/atoms/tag/tag.html" with item=tag small=True tag_link_base=blog_index_url %}
+                {% endfor %}
+            </div>
+        </div>
 
-        {% if page.read_time %}
-            / {{ page.read_time }} min read
-        {% endif %}
-    </div>
+        <div class="blog-meta">
+            <span>Insight</span>
+            <span aria-hidden="true" class="blog-meta__divider">/</span>
+
+            <time datetime="{{ item.date|date:'Y-m-d' }}">{{ page.date }}</time>
 
-    {% include_block page.body %}
+            {% if page.read_time %}
+                <span aria-hidden="true" class="blog-meta__divider">/</span>
+                {{ page.read_time }} min read
+            {% endif %}
+        </div>
 
-    {% if page.related_blog_posts %}
-        {% include "patterns/atoms/section-title/section-title.html" with title="More"%}
+        {% include_block page.body %}
 
-        {% for post in page.related_blog_posts %}
-            {% include "patterns/molecules/blog-item/blog-item.html" with item=post %}
-        {% endfor %}
-    {% endif %}
+        {% if page.related_blog_posts %}
+            {% include "patterns/atoms/section-title/section-title.html" with title="More" aria_label="Related blog posts" classes="section-title--related-posts" %}
 
+            <ul class="streamfield__related-posts">
```

Comment:
We may want to move this to an include as we will also have a streamfield block that looks very simiar - but maybe we can do that next sprint.